src/tests/containerizer/cgroups2_tests.cpp

```diff
@@ -177,6 +179,35 @@ TEST_F(Cgroups2Test, ROOT_CGROUPS2_CpuStats)
   ASSERT_SOME(cgroups2::cpu::stats(TEST_CGROUP));
 }
 
+
+TEST_F(Cgroups2Test, ROOT_CGROUPS2_EnableAndDisable)
+{
+  ASSERT_SOME(cgroups2::create(TEST_CGROUP));
```

Comment:
we want to call enable_controllers here to ensure the root cgroup has the cpu controller enabled, right?